### PR TITLE
Scalar render parfix

### PR DESCRIPTION
**Pull request by nicolemarsaglia**
Fix what I broke.

### CODE REVIEW

**Comment by nicolemarsaglia** [comment]
This is to undo breaking scalar renderer in parallel.

File: src/vtkh/compositing/PayloadCompositor.cpp [hunk 1]
```diff
@@ -47,16 +47,19 @@ PayloadCompositor::AddImage(PayloadImage &image)
 PayloadImage
 PayloadCompositor::Composite()
 {
+  std::cerr << "In PAYLOAD COMPOSITE" << std::endl;
   assert(m_images.size() != 0);
-
+  std::cerr << "image size: " << m_images.size() << std::endl;
```
Comment (mclarsen) [inline src/vtkh/compositing/PayloadCompositor.cpp]: Do you need the print statements?

**Review by mclarsen — CHANGES_REQUESTED** [review]
Without the prints we are good.

File: src/vtkh/compositing/PayloadCompositor.cpp (hunk 1 above)
Comment (nicolemarsaglia) [reply src/vtkh/compositing/PayloadCompositor.cpp]: Nope. my bad.

Comment (nicolemarsaglia) [reply src/vtkh/compositing/PayloadCompositor.cpp]: I didn't mean to push this file.

**Review by mclarsen — APPROVED** [review]

**Comment by mclarsen** [comment]
The CI wont pass because we have detected blt time travel. You need to `git submodule update` to get the current version. Also you could
```
cd src/blt
git checkout 86a7d6b5ee93d8b7b3233a3e54a4f06f8d8308d3
cd ../..
git push
```

**Pull request merged**